components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/AwesomeBarOrientation.kt

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.compose.browser.awesomebar
+
+/**
+ * The orientation of the AwesomeBar, whether it's oriented to the bottom or the top.
+ */
+enum class AwesomeBarOrientation {
```

Comment:
nit: should it be orientation or location?

Reply:
I think this refers to the orientation of the autocomplete suggestion "arrow": `mozac_ic_edit_suggestion`. It should be orientated to the top if the toolbar is at the top and, and to the bottom if the toolbar is at the bottom. My 2 cents, we should lean towards naming that refers to the composable itself, not other elements, so I would actually prefer this name.